pkg/config/default.go

```diff
@@ -371,6 +371,14 @@ func defaultConfigFromMemory() (*EngineConfig, error) {
 		"/usr/local/bin/conmon",
 		"/usr/local/sbin/conmon",
 		"/run/current-system/sw/bin/conmon",
+		"/usr/libexec/podman/conmonrs",
```

Comment:
Could we add a new field `ConmonrsPath` field to the config and have a separate list here?  It's effectively a new tool.

Reply:
Sure thing, I will get started on that today.